Doc/library/sqlite3.rst

```diff
@@ -2409,20 +2410,15 @@ Here's an example of one returning a :class:`dict`:
    ...     print(row)
    {'a': 1, 'b': 2}
 
-Here's an example of a factory that return a :class:`~collections.namedtuple`.
+The following row factory returns a :class:`~collections.namedtuple`.
 
 .. testcode::
 
    from collections import namedtuple
 
-   def _get_fields(cursor):
-        return (col[0] for col in cursor.description)
-
-   def _make_cls(fields):
-       return namedtuple("Row", fields)
-
    def namedtuple_factory(cursor, row):
-       cls = _make_cls(_get_fields(cursor))
+       fields = [col[0] for col in cursor.description]
```

Comment:
Since you have the space, maybe be a touch clearer with your variable names (especially for non-native speakers)?

```suggestion
       fields = [column[0] for column in cursor.description]
```

Reply:
Yes, that's better

Reply:
And this one?